<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fChonThuMuc</class>
 <widget class="QDialog" name="fChonThuMuc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFBDA;
color:black;
font: 500 9pt &quot;Roboto Medium&quot;;
</string>
  </property>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Medium</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:darkred;
border-radius:20px;
color:white;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Đóng</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(53, 120, 229);
color:white;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/C:/Users/quan2/Downloads/up-arrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/C:/Users/quan2/Downloads/down-arrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="dtgvAcc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string>(Click đúp vào dòng để chọn)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCountChoose">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Medium</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;
border-width: 0; /* Set the width of the border */</string>
   </property>
   <property name="text">
    <string>Tất cả:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCountTotal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
color:darkred;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Medium</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;
border-width: 0; /* Set the width of the border */
</string>
   </property>
   <property name="text">
    <string>Đã chọn:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox1">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>35</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
